### DISCUSSION TITLE
How to set the game to immersive mode when use adobe air sdk?

### DISCUSSION THREAD

**Discussion by pengyangjin** [airsdk/Adobe-Runtime-Support#1280] [Q&A]
Now full screen mobile phones are becoming more and more common.Making games with adobe air will trigger a full screen gesture when clicking on the edge area.How can I put the game in immersive mode?

**Comment by 2jfw** [comment]
There is a commercial but up-to-date ANE from distriqt:
https://airnativeextensions.com/extension/com.distriqt.Application

There are also two free ANEs which have _not_ been maintained for a long time so you will likely run into problems with them sooner or later:
https://github.com/mesmotronic/air-ane-fullscreen
https://github.com/mesmotronic/air-ane-immersivemode

**EDIT:** By the way, you falsely tagged your question as "Bug"..

**Comment by marchbold** [comment]
If you are talking about the iOS edge gestures, there is also the SystemGestures ANE that allows you to defer them (i.e. the home bar gets hidden and the user has to swipe up to display it and then again to go home). 

https://github.com/distriqt/ANE-SystemGestures

(I'll transfer this to a discussion)
